### PR TITLE
DOCS-7624-applied template

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/apply-configure-xml-threat-task.adoc [hunk 1]
```diff
@@ -9,31 +9,34 @@ endif::[]
 >s|Policy Name|XML Threat Protection
 >s|Summary|Protects against malicious XML in API requests
 >s|Category|Security
->s|Since Mule Version|3.8.0
+>s|Available from Mule version|3.8.0
```
Comment (kbostrom) [inline modules/ROOT/pages/apply-configure-xml-threat-task.adoc:12]: "First Mule version available"?

File: modules/ROOT/pages/apply-configure-xml-threat-task.adoc [hunk 2]
```diff
@@ -9,31 +9,34 @@ endif::[]
...
 >s|Returned Status Codes|400 - Bad Request
 |===
 
-== Purpose
 
-XML requests are susceptible to attacks characterized by unusual inflation of elements, attributes and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The XML threat protection policy helps protect your applications from such intrusions.
+Applications processing XML requests are susceptible to attacks characterized by unusual inflation of elements, attributes, and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The XML threat protection policy helps protect your applications from such intrusions.
```
Comment (kbostrom) [inline modules/ROOT/pages/apply-configure-xml-threat-task.adoc]:
```suggestion
Applications processing XML requests are susceptible to attacks characterized by unusual inflation of elements, attributes, and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic increases in the size of the application data often signal a security problem. The XML threat protection policy helps protect your applications from such intrusions.
```
Is "the XML threat protection policy" the same as "the JSON Threat Protection policy"?
If so, try to stick with just one name.
If not...be sure that the first is a common noun ("a XML threat protection policy") or a proper noun, like the JSON version.

File: modules/ROOT/pages/apply-configure-xml-threat-task.adoc [hunk 3]
```diff
@@ -9,31 +9,34 @@ endif::[]
...
 [width="100%", cols="2,6,1,1"]
 |===
 s|Field s|Description s|Default s|Required
-|Maximum Node Depth|Specifies the maximum node depth allowed in the XML ^|0 ^|false
-|Maximum Attribute Count Per Element|Specifies the maximum number of attributes allowed for any element. Note that attributes used for defining namespaces are not counted ^|0 ^|false
-|Maximum Child Count|Specifies a limit on the maximum number of children allowed for any element in the XML document ^|0 ^|false
-|Maximum Text Length|Specifies a limit on the maximum length, in characters, of any text nodes present in the XML document ^|0 ^|false
-|Maximum Attribute Length|Specifies a limit on the maximum length, in characters, of any attributes for any element in the XML document ^|0 ^|false
-|Maximum Comment Length|Specifies a limit on the maximum number of comment characters in the XML document ^|0 ^|false
+|Maximum Node Depth|Specifies the maximum node depth of an XML document ^|0 ^|false
```
Comment (kbostrom) [inline modules/ROOT/pages/apply-configure-xml-threat-task.adoc]: period
```suggestion
|Maximum Node Depth|Specifies the maximum node depth of an XML document. ^|0 ^|false
```

File: modules/ROOT/pages/apply-configure-xml-threat-task.adoc [hunk 4]
```diff
@@ -9,31 +9,34 @@ endif::[]
...
+|Maximum Attribute Count Per Element|Specifies the maximum number of attributes in an element. Note that attributes used for defining namespaces are not counted ^|0 ^|false
```
Comment (kbostrom) [inline modules/ROOT/pages/apply-configure-xml-threat-task.adoc]: ditto
```suggestion
|Maximum Attribute Count Per Element|Specifies the maximum number of attributes in an element. Note that attributes used for defining namespaces are not counted. ^|0 ^|false
```

Comment (kbostrom) [reply modules/ROOT/pages/apply-configure-xml-threat-task.adoc]: Please supply a period at the end of each of the following description texts, as in the previous two edits.

File: modules/ROOT/pages/apply-configure-xml-threat-task.adoc [hunk 5]
```diff
@@ -9,31 +9,34 @@ endif::[]
...
+|Maximum Child Count|Specifies the maximum number of children of an element in the XML document ^|0 ^|false
+|Maximum Text Length|Specifies the maximum length (in characters) of text nodes in the XML document ^|0 ^|false
+|Maximum Attribute Length|Specifies the maximum length (in characters) of an attribute in the XML document ^|0 ^|false
+|Maximum Comment Length|Specifies the maximum number of comment characters in the XML document ^|0 ^|false
 |===
 
 NOTE: Configuring a value of -1 means unlimited.
```
Comment (kbostrom) [inline modules/ROOT/pages/apply-configure-xml-threat-task.adoc]:
```suggestion
NOTE: A value of -1 indicates that the field value has no limits.
```

File: modules/ROOT/pages/apply-configure-xml-threat-task.adoc [hunk 6]
```diff
@@ -9,31 +9,34 @@ endif::[]
...
 
 == Example
 
+The following screenshot shows an example of the parameters configured for the XML Threat Protection policy:
```
Comment (kbostrom) [inline modules/ROOT/pages/apply-configure-xml-threat-task.adoc:40]: Ah! The policy name is init. capped here, so likely should be above.

**Review by kbostrom — APPROVED** [review]
Done

File: modules/ROOT/pages/apply-configure-xml-threat-task.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/apply-configure-xml-threat-task.adoc:12]: We have been doing "Available from Mule version" in other policies. Should we stick to that?

File: modules/ROOT/pages/apply-configure-xml-threat-task.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/apply-configure-xml-threat-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/apply-configure-xml-threat-task.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/apply-configure-xml-threat-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/apply-configure-xml-threat-task.adoc (hunk 4 above)
Comment (priyaayyar) [reply modules/ROOT/pages/apply-configure-xml-threat-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/apply-configure-xml-threat-task.adoc (hunk 5 above)
Comment (priyaayyar) [reply modules/ROOT/pages/apply-configure-xml-threat-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/apply-configure-xml-threat-task.adoc (hunk 6 above)
Comment (priyaayyar) [reply modules/ROOT/pages/apply-configure-xml-threat-task.adoc:40]: Yes changed that! Done! Thanks!

**Pull request merged**